specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2021-05-01-preview/mysql.json

```diff
@@ -0,0 +1,2606 @@
+          "description": "Restore point creation time (ISO8601 format), specifying the time to restore from.",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "replicationRole": {
+          "$ref": "#/definitions/ReplicationRole",
+          "description": "The replication role."
+        },
+        "replicaCapacity": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 0,
+          "readOnly": true,
+          "description": "The maximum number of replicas that a primary server can have."
+        },
+        "state": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The state of a server.",
+          "enum": [
+            "Ready",
+            "Dropping",
+            "Disabled",
+            "Starting",
+            "Stopping",
+            "Stopped",
+            "Updating"
+          ],
+          "x-ms-enum": {
+            "name": "ServerState",
+            "modelAsString": true
+          }
+        },
+        "fullyQualifiedDomainName": {
+          "type": "string",
+          "description": "The fully qualified domain name of a server.",
+          "readOnly": true
+        },
+        "storage": {
+          "$ref": "#/definitions/Storage",
+          "description": "Storage related properties of a server."
+        },
+        "backup": {
+          "$ref": "#/definitions/Backup",
+          "description": "Backup related properties of a server."
+        },
+        "highAvailability": {
+          "$ref": "#/definitions/HighAvailability",
+          "description": "High availability related properties of a server."
+        },
+        "network": {
+          "$ref": "#/definitions/Network",
+          "description": "Network related properties of a server.",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "maintenanceWindow": {
+          "$ref": "#/definitions/MaintenanceWindow",
+          "description": "Maintenance window of a server.",
+          "x-ms-mutability": [
+            "update",
+            "read"
+          ]
+        }
+      },
+      "description": "The properties of a server."
+    },
+    "Server": {
+      "type": "object",
+      "properties": {
+        "identity": {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/Identity",
+          "description": "The Azure Active Directory identity of the server."
+        },
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "The SKU (pricing tier) of the server."
+        },
+        "properties": {
+          "$ref": "#/definitions/ServerProperties",
+          "x-ms-client-flatten": true,
+          "description": "Properties of the server."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
```

Comment:
`TrackedResource` already defined `tags`

Reply:
thanks, removed tags in new commit.